src/test/kotlin/io/libp2p/core/security/secio/SecIoSecureChannelTest.kt

```diff
@@ -69,6 +70,10 @@ class SecIoSecureChannelTest {
         Assertions.assertEquals("Hello World from 1", rec2)
         Assertions.assertEquals("Hello World from 2", rec1)
     }
+
+    companion object {
+        private val logger = LogManager.getLogger(SecIoSecureChannelTest::class.java)
+    }
```

Comment:
I'd better declare it in one line, like: `private const val logger = `

Reply:
Can't do `const` on non-primitives.

But are you ok to keep them in the companion objects so that they remain as singletons?